Citrix-Workspace/tools/ChocolateyInstall.ps1

```diff
@@ -1,15 +1,38 @@
 ## Template VirtualEngine.Build ChocolateyInstall.ps1 file for EXE/MSI installations
 
-$releaseUri = 'https://www.citrix.com/downloads/workspace-app/legacy-workspace-app-for-windows/workspace-app-for-windows-1903.html'
-$downloadLink = 'https:{0}' -f ((Invoke-WebRequest -Uri $releaseUri).Links | Where-Object { $_.href -eq '#ctx-dl-eula' } | Select-Object -ExpandProperty 'rel')
+$releaseUri = "https://www.citrix.com/downloads/workspace-app/legacy-workspace-app-for-windows/workspace-app-for-windows-1903.html"
+$checksum = "086DBE728BFBA427D1DF28C3A7ADEB072CD3878758084A547879759EE36064B5"
+
+$READYSTATE_READY = 4
```

Comment:
ℹ️ This came from the example at https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/automatic/wps-office-free/update_helper.ps1